package.json

```diff
@@ -103,5 +103,39 @@
     "packages": [
       "packages/ckeditor5-dev/packages/ckeditor5-dev-tests"
     ]
+  },
+  "resolutions": {
+    "ckeditor5": "35.1.0",
+    "@ckeditor/ckeditor5-autoformat": "35.1.0",
+    "@ckeditor/ckeditor5-basic-styles": "35.1.0",
+    "@ckeditor/ckeditor5-block-quote": "35.1.0",
+    "@ckeditor/ckeditor5-clipboard": "35.1.0",
+    "@ckeditor/ckeditor5-code-block": "35.1.0",
+    "@ckeditor/ckeditor5-core": "35.1.0",
+    "@ckeditor/ckeditor5-editor-classic": "35.1.0",
+    "@ckeditor/ckeditor5-editor-decoupled": "35.1.0",
+    "@ckeditor/ckeditor5-engine": "35.1.0",
+    "@ckeditor/ckeditor5-enter": "35.1.0",
+    "@ckeditor/ckeditor5-essentials": "35.1.0",
+    "@ckeditor/ckeditor5-heading": "35.1.0",
+    "@ckeditor/ckeditor5-horizontal-line": "35.1.0",
+    "@ckeditor/ckeditor5-image": "35.1.0",
+    "@ckeditor/ckeditor5-inspector": "4.0.0",
+    "@ckeditor/ckeditor5-link": "35.1.0",
+    "@ckeditor/ckeditor5-list": "35.1.0",
+    "@ckeditor/ckeditor5-markdown-gfm": "35.1.0",
+    "@ckeditor/ckeditor5-mention": "35.1.0",
+    "@ckeditor/ckeditor5-paragraph": "35.1.0",
+    "@ckeditor/ckeditor5-paste-from-office": "35.1.0",
+    "@ckeditor/ckeditor5-remove-format": "35.1.0",
+    "@ckeditor/ckeditor5-select-all": "35.1.0",
+    "@ckeditor/ckeditor5-table": "35.1.0",
+    "@ckeditor/ckeditor5-typing": "35.1.0",
+    "@ckeditor/ckeditor5-theme-lark": "35.1.0",
+    "@ckeditor/ckeditor5-ui": "35.1.0",
+    "@ckeditor/ckeditor5-undo": "35.1.0",
+    "@ckeditor/ckeditor5-upload": "35.1.0",
+    "@ckeditor/ckeditor5-utils": "35.1.0",
+    "@ckeditor/ckeditor5-widget": "35.1.0"
```

Comment:
AFAIU this change should solve the duplicated modules problem with mermaid and possibly other packages, right? If so, it deserves an issue to not go unnoticed/unclear.

Reply:
This was a part of the work done here -> [https://github.com/ckeditor/github-writer/pull/364/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R106-R139.](https://github.com/ckeditor/github-writer/pull/364/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R106-R139.) I've copied it locally initially because we were working together on this with Piotrek that's why it's here. Potentially we could simply merge the `ghw/epic/367-version-3510-bump` branch? And discuss if/how we should document this change so it's not unnoticed/unclear as you've said.